getting_started/devpreview_faq.adoc

```diff
@@ -165,6 +165,10 @@ volume claim (PVC) or an EmptyDir at this time.
 * Only the Source-to-Image (S2I) build strategy is allowed for any build
 configurations imported into your project.
 
+[[devpreview-access-internal-registry]]
+HOW DO I ACCESS THE INTERNAL REGISTRY?::
+The internal registry is located at *_registry.preview.openshift.com_*.
```

Comment:
:+1: Could link to this too: managing_images.html#accessing-the-internal-registry